Sources/Plasma/NucleusLib/pnInputCore/plInputMap.cpp

```diff
@@ -49,6 +49,10 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 
 #include "plResMgr/plLocalization.h"
 
+#ifdef HS_BUILD_FOR_MACOS
+#include <Carbon/Carbon.h>
+#endif
```

Comment:
I don't think this needed here anymore (since we're using Plasma's `KEY_X` values now rather than Carbon's `kVK_ANSI_X` ones)